pages/docs/security.mdx

```diff
@@ -0,0 +1,121 @@
+---
+description: Survey of common security problems facing LLM-based applications and how to use Langfuse to trace, prevent, and evaluate security risks.
+---
+
+# Security with Langfuse
+
+There are a host of potential security risks involved with LLM-based applications, such as prompt injection, leakage of personally identifiable information (PII), or harmful prompts. Langfuse can be used to monitor and protect against these security risks.
+
```

Comment:
I'd add a second paragraph here to outline how Langfuse + security packages are used in tandem

langfuse
- track security mechanisms to debug and observe whether they work as expected
- track security metrics
- fully async, does not interrupt program execution